Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ReflectometryILLPreprocess.py

```diff
@@ -155,21 +156,21 @@ def PyInit(self):
                                                      validator=WorkspaceUnitValidator('TOF'),
                                                      optional=PropertyMode.Optional),
                              doc='An input workspace (units TOF) if no Run is specified.')
-        self.declareProperty(ITableWorkspaceProperty(Prop.BEAM_POS_WS,
-                                                     defaultValue='',
-                                                     direction=Direction.Input,
-                                                     optional=PropertyMode.Optional),
-                             doc='A beam position table corresponding to InputWorkspace.')
-        self.declareProperty(Prop.BEAM_ANGLE,
-                             defaultValue=Property.EMPTY_DBL,
-                             doc='A user-defined beam angle (unit degrees).')
-        self.declareProperty(name=Prop.BEAM_CENTRE,
-                             defaultValue=Property.EMPTY_DBL,
-                             doc='A workspace index corresponding to the beam centre.')
         self.declareProperty(MatrixWorkspaceProperty(Prop.OUTPUT_WS,
                                                      defaultValue='',
                                                      direction=Direction.Output),
                              doc='The preprocessed output workspace (unit wavelength), single histogram.')
+        self.declareProperty(Prop.TWO_THETA,
+                             defaultValue=Property.EMPTY_DBL,
+                             doc='A user-defined scattering angle 2 theta (unit degrees).')
+        self.declareProperty(name=Prop.LINE_POSITION_INPUT,
+                             defaultValue=Property.EMPTY_DBL,
+                             doc='A workspace index corresponding to the beam centre between 0.0 and 255.0.')
+        self.declareProperty(MatrixWorkspaceProperty(Prop.DIRECT_LINE_WORKSPACE,
+                                                     defaultValue='',
+                                                     direction=Direction.Input,
+                                                     optional=PropertyMode.Optional),
+                             doc='A pre-processed direct beam workspace (sample log entries).')
```

Comment:
Users might not understand what the cryptic `(sample log entries)` means. I suggest we just remove it. Or use something like `'A pre-processed direct beam workspace for detector angle calibration.'`.